articles/130_ros_time.md

```diff
@@ -126,9 +130,9 @@ However, if a client library chooses to not use the shared implementation then i
 
 In each implementation will provide `Time`, `Duration`, and `Rate` datatypes, for all three time source abstraction..
 The `Duration` will support a `sleep_for` function as well as a `sleep_until` method.
-The implementation will also provide a `Timer` object which will provide periodic callback functionality for all the abstractions.
+The implementation will also provide a `Timer` object which will provide periodic callback functionality for all the abstractions. It will be microsecond-accurate.
```

Comment:
Why microsecond? At the moment our time structure uses nanoseconds as storage. I'm hesitant to try and make any guarantees about the accuracy of the timers, since they are often affected by other timers and/or work by default.